pyproject.toml

```diff
@@ -15,19 +15,19 @@ dependencies = [
     "ruff==0.8.0",
     "pre-commit>=4.0.1,<5",
     "matplotlib==3.9.*",
-    "lightning>=2.4.0,<3",
+    "lightning==2.5.0.post0",
```

Comment:
I think this is kicking the can down the road no? At some point we will want to update, and when we do the ckpt path stuff will break again